composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require" : {
         "php": ">=5.3",
-        "symfony/framework-bundle": "2.3||^3.0"
+        "symfony/framework-bundle": "^2.3||^3.0"
```

Comment:
Please change it to `>=2.3` , which should be acceptable for Symfony versions - 2.3 and higher, as this bundle seems to be common for both 2.* and 3.*